include/wx/charts/wxchartslabel.h

```diff
@@ -55,6 +59,18 @@ class wxChartsLabel : public wxChartsElement
     wxChartsLabel(const wxString &text, wxDouble width,
         wxDouble height, const wxChartsLabelOptions &options);
 
+public:
+    bool operator==(const wxChartsLabel & rhs)
```

Comment:
Can we remove the equality operators? They don't compare the entire state of the class and there are already accessors for the text so users can simply do label.GetText() == otherLabel.GetText(). This is maybe slightly longer but more explicit about exactly what is being compared.

Reply:
Without `operator!=`, compile error as below
```
\path\to\include\wx/vector.h(452): error C2678: binary '!=': no operator found which takes a left-hand operand of type 'wxChartsLabel' (or there is no acceptable conversion)
```